### ISSUE TITLE
Could not get it to work

### ISSUE THREAD

**Issue by lirankor** [moh3a/ae_sdk#9]
I have tried to implement without success:

```
// Import the SDK
import { DropshipperClient } from "ae_sdk";

// Initialize the SDK (replace with actual parameters if needed)
const client = new DropshipperClient({
    app_key: 'XXXXX',
    app_secret: 'XXXXXXXX',
    session: "oauth_access_token"
});

async function getProductData(productId) {
    try {
      const productData = await client.productDetails({
        product_id: productId,
        ship_to_country: 'US',
        target_currency: 'USD',
        target_language: 'en',
      });
      console.log(productData);
    } catch (error) {
      console.error(`An error occurred: ${error.message}`);
    }
  }

// Example usage
const productId = '3256806022897905';
getProductData(productId);
```

Using correct key / secret

```
{
  ok: false,
  message: 'Bad request',
  error_response: {
    type: 'ISV',
    code: 'IllegalAccessToken',
    msg: 'The specified access token is invalid or expired',
    request_id: 'xxxx'
  },
  request_id: 'xxxx'
}
```

Did I miss anything?

**Comment by moh3a** [comment]
It seems that the issue is related to the access token you passed to client.session. According to the error message, the access token might be invalid or expired.

Please verify the validity of the token you're using. For more information on access tokens and how to resolve this issue, you can refer to the [official documentation here](https://openservice.aliexpress.com/doc/doc.htm?spm=a2o9m.11193487.0.0.372eee0clw47N8&nodeId=27493&docId=118729#/?docId=1364).

**Comment by lirankor** [comment]
Thank you for responding!

Testing with the same credencials on the playground:
https://openservice.aliexpress.com/app/index.htm?spm=a2o9m.11193495.0.0.3b066ec6kqZrOk#/api/test?id=11980&appkey=501702&_k=ezrtjq
this, works well after genrating "Access Token" using the "Get Token" button. could it be that the access token is not being generated when creating a new client? 
According to this:
https://openservice.aliexpress.com/doc/doc.htm?spm=a2o9m.11193531.0.0.53483b536y2b32&nodeId=27493&docId=118729#/?docId=1364
We need to first authenticate, step 4, in order to create access token.

**Comment by moh3a** [comment]
> could it be that the access token is not being generated when creating a new client?

Yes, the client does not generate an access token, it is a different workflow detailed in the docs and you pass it to the client
